src/routes/api/ranking.route.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const rankingController = require('../../controllers/ranking.controller');
+const { auth } = require('../../middlewares/authorization');
+
+const router = express.Router();
+
+router.get('/top_rankings', auth, rankingController.get_top_rankings);
```

Comment:
can you change this to `/featured`? it feels redundant to do`/rankings/top_rankings` as rankings is defined in both. Also,I prefer to use hypens (`-`) instead of `_` when designing endpoints. 

yea so lets do `/rankings/featured` its more defined that way!

Reply:
ok i do that